mjs_firmware.ino

```diff
@@ -93,7 +91,7 @@ uint8_t const LUX_HIGH_PIN = 5;
 
 // setup timing variables
 uint32_t const UPDATE_INTERVAL = 900000;
-uint32_t const GPS_TIMEOUT = 120000;
+uint32_t const GPS_TIMEOUT = 360000;
```

Comment:
You are also changing the GPS timeout here. This should at least be moved into a separate commit, I think (I'm happy to do that if that's too much git magic for you :-p). However, I wonder whether the stability improvements you are seeing are due to the initialization changes, or perhaps due to this timeout increase. Ideally, I'd not increase the timeout too much, since that can potentially increase battery drain significantly when reception is bad (OTOH, the alternative is probably to have no position at all, which isn't great either).

Reply:
@matthijskooijman This is mistake. I changed this value on my station at some point to see if allowing the GPS some more time to fix would improve things. As far as I can tell, it did not. I will revert this change.
```suggestion
uint32_t const GPS_TIMEOUT = 120000;
```